### ISSUE TITLE
Doubt about the final saved parameter values of the model for the private test 

### ISSUE THREAD

**Issue by zxfly2** [WuJie1010/Facial-Expression-Recognition.Pytorch#27]
First of all, thank you for sharing the code. But I have a small question. The final saved parameter values of the model for the private test correspond to the parameters of the convolution network when the model has the highest detection accuracy on the (Private test)dataset. However, in practical applications, the test set cannot be obtained in advance, so the accuracy of the test set is not convincing. I think that the validation set (Public Test) should be used as a reference standard. The final parameter value of the model used for the test set(private test) is corresponding to the parameters of the convolution network when the model has the highest detection accuracy on the validation set(Private test). I think this is more reasonable.Private test is only used during the final test.

**Comment by WuJie1010** [comment]
Yes, you are correct. But we also find that the best ckpt in validation set also performs best in test set.

**Comment by zxfly2** [comment]
Thank you for your reply. But there is a difference between the validation set and the test set, so the best ckpt of validation set may be not corresponding to the best performance on test set.And after trying for several times,I find that the model performs about 72.6%(not reach 73%) on test set with the best ckpt of validation set.I tryed for 5 times and took the average value of the results.I venture to think that using this value as the final test accuracy may be more rigorous.

**Comment by WuJie1010** [comment]
Yes, you are correct.

**Comment by zxfly2** [comment]
Ok, thank you for your prompt reply.
